MIGRATION.md

```diff
@@ -537,3 +539,125 @@ import { Grid, GridItem } from '@contentful/f36-components';
   <GridItem />
 </Grid>;
 ```
+
+### Tabs
+
+The API for Tabs components was changed. First of all, there is no `role` prop anymore, Tabs could be only used with role `"tabs"`. This is how the new API looks now:
+
+```tsx static=true
+import { Tabs } from '@contentful/f36-components';
+
+<Tabs defaultTab="first">
+  <Tabs.List variant="vertical-divider">
+    <Tabs.Tab panelId="first">First</Tabs.Tab>
+    <Tabs.Tab panelId="second">Second</Tabs.Tab>
+    <Tabs.Tab panelId="third">Third</Tabs.Tab>
+  </Tabs.List>
+  <Tabs.Panel id="first">Content for the first tab</Tabs.Panel>
+  <Tabs.Panel id="second">Content for the second tab</Tabs.Panel>
+  <Tabs.Panel id="third">Content for the third tab</Tabs.Panel>
+</Tabs>;
+```
+
+It's also possible to use `Tabs` as a controlled component. Here is the example:
+
+```tsx static=true
+import { Tabs } from '@contentful/f36-components';
+
+() => {
+  const [currentTab, setCurrentTab] = React.useState('first');
+  return (
+    <Tabs currentTab={currentTab} onTabChange={setCurrentTab}>
+      <Tabs.List>
+        <Tabs.Tab panelId="first">First</Tabs.Tab>
+        <Tabs.Tab panelId="second">Second</Tabs.Tab>
+        <Tabs.Tab panelId="third">Third</Tabs.Tab>
+      </Tabs.List>
+      <Tabs.Panel id="first">content first tab</Tabs.Panel>
+      <Tabs.Panel id="second">content second tab</Tabs.Panel>
+      <Tabs.Panel id="third">content third tab</Tabs.Panel>
+    </Tabs>
+  );
+};
+```
+
+#### How to migrate your Grid components
+
+With the sagnificant changes to Tabs API, we don't have a codemod to migrate this component. This will have to be done manually. From this:
```

Comment:
```suggestion
Because of the significant changes done to the Tabs API, you cannot run a codemod to migrate this component. You must manually migrate the Tabs component by replacing:
```